services/storage/docker-compose.yml

```diff
@@ -22,7 +22,7 @@ services:
       - NEOFS_NODE_KEY=/01.key
       - NEOFS_NODE_ADDRESS=s01.${LOCAL_DOMAIN}:8080
       - NEOFS_GRPC_ENDPOINT=s01.${LOCAL_DOMAIN}:8080
-      - NEOFS_NODE_ATTRIBUTE_0=/Location:Europe/Country:Russia/City:Moscow
+      - NEOFS_NODE_ATTRIBUTE_0=/Locode:RU MSK
```

Comment:
Do we really need the `/` prefix?

Reply:
No. We have not defined attribute parsing rules, however both `/Locode:RU MSK` and `Locode:RU MSK` going to be parsed the same way (see [tests](https://github.com/nspcc-dev/neofs-node/blob/master/pkg/util/attributes/parser_test.go))

Reply:
However I would like to define LOCODE as a separate node configuration value instead of raw attribute.

Reply:
No, both `/Attribute:X` and `Attribute:X` parsed the same way. Fixed.